Sources/BaggageContext/ContextLogHandler.swift

```diff
@@ -86,7 +103,7 @@ public struct BaggageMetadataLogHandler: LogHandler {
 
     private func baggageAsMetadata() -> Logger.Metadata {
         var effectiveMetadata: Logger.Metadata = [:]
-        self.context.forEach { key, value in
+        self.baggage.forEachBaggageItem { key, value in
```

Comment:
This was renamed on purpose; if a library decides to adopt `BaggageProtocol` it would gain this `forEach` which would read weird, e.g. vapor would:

```
// request.forEach { baggageKey, baggageValue in 
```

which reads like nonsense; as such, we make the API more explicit (which is also following other Tracing APIs which have Baggage and Baggage items like this).